composer.json

```
@@ -3,13 +3,12 @@
     "type": "symfony-bundle",
     "description" : "Provide APCu support",
     "require": {
-        "php"                            : ">=5.5",
+        "php"                            : ">=7.2",
```

```suggestion
        "php"                            : ">=7.4",
```

since we're doing a breaking change by dropping php5.5, 5.6, 7.0 and 7.1, we could also drop other unsupported PHP versions, WDYT?